codes/quantum/qubits/subsystem/subsystem_sparse.yml

```diff
@@ -0,0 +1,37 @@
+code_id: subsystem_sparse
```

Comment:
Please make stub entry for code_id: sparse_stabilizer, and implement the following tree of parents (code to edit <-- parent):
qldpc <-- sparse_stabilizer
this code <-- sparse_stabilizer
sparse_stabilizer <-- stabilizer